pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -30,6 +30,8 @@ require("page.css");
  * Expected props:
  *  - cancel_clicked optional
  *     Callback called when the dialog is canceled
+ *  - cancel style
+ *     Defaults to cancel
```

Comment:
How about something like `css class used for the cancel button, defaults to 'cancel'`